package.json

```diff
@@ -26,13 +26,24 @@
     "zone.js": "~0.11.4"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "^14.0.5",
+    "@angular-devkit/build-angular": "^14.2.2",
+    "@angular-eslint/builder": "14.0.4",
+    "@angular-eslint/eslint-plugin": "14.0.4",
+    "@angular-eslint/eslint-plugin-template": "14.0.4",
+    "@angular-eslint/schematics": "14.0.4",
+    "@angular-eslint/template-parser": "14.0.4",
     "@angular/cli": "^14.0.5",
     "@angular/compiler-cli": "^14.0.5",
     "@angular/language-service": "^14.0.5",
     "@types/jasmine": "~3.6.0",
     "@types/node": "^12.11.1",
+    "@typescript-eslint/eslint-plugin": "5.36.2",
+    "@typescript-eslint/parser": "5.36.2",
     "codelyzer": "^6.0.0",
```

Comment:
Codelyzer was intended to be used in conjunction with TSLint. So, we can get rid of this dependency.